### PR TITLE
Intercom Provider

### PR DESCRIPTION
**Pull request by bryceadams**

### CODE REVIEW

**Comment by faustbrian** [comment]
The Provider also needs to be added to https://github.com/SocialiteProviders/Providers/blob/master/build/split.sh

File: src/Intercom/Provider.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'INTERCOM';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [''];
```
Comment (faustbrian) [inline src/Intercom/Provider.php]: The `$scopes` array can be removed as it is just an empty array like already defined here https://github.com/laravel/socialite/blob/3.0/src/Two/AbstractProvider.php#L62.

**Comment by bryceadams** [comment]
👍

**Comment by faustbrian** [comment]
Thanks! Will be added to packagist in a few minutes.

**Comment by bryceadams** [comment]
Thanks for the fast turnaround!

**Comment by faustbrian** [comment]
https://packagist.org/packages/socialiteproviders/intercom

**Pull request merged**